keyboards/oddball/rules.mk

```
@@ -33,7 +33,10 @@ HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
 
 DEBOUNCE_TYPE = eager_pk
 SPLIT_KEYBOARD = yes
-
-# Add trackball support
 POINTING_DEVICE_ENABLE = yes
-SRC += adns.c
+
+DEFAULT_FOLDER = oddball/v2
```

Setting this to v2 will mean it is considered a "breaking change", and require it to go via develop.
```suggestion
DEFAULT_FOLDER = oddball/v1
```

---

Done.